Fixes #486 (#490)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Entities/IfcArchElementType/Documentation.md

> The element type _IfcArchElementType_ defines commonly shared information for occurrences of arch elements. Arch elements are building elements which represents unitary curved structures.The set of shared information may include:

Reviewer: ```suggestion
The element type _IfcArchElementType_ defines commonly shared information for occurrences of arch elements. Arch elements are building elements which represents unitary curved structures. The set of shared information may include:
```

> The element type _IfcArchElementType_ defines commonly shared information for occurrences of arch elements. Arch elements are building elements which represents unitary curved structures. The set of shared information may include:

Reviewer 2: ```suggestion
The element type _IfcArchElementType_ defines commonly shared information for occurrences of arch elements. Arch elements are building elements which represent unitary curved structures. The set of shared information may include:
```

* common properties within shared property sets
* common material information
* common profile definitions
* common shape representations

It is used to define an arch element specification that is common to all occurrences of that arch element type. Arch element types may be exchanged without being already assigned to occurrences.

> Occurrences of the _IfcFillElementType_ within building models are represented by instances of _IfcFillElement_.

Reviewer: *Fill*?

Reviewer 3: ```suggestion
Occurrences of the _IfcArchElementType_ within building models are represented by instances of _IfcArchElement_.
```
